tests/linen/summary_test.py

```diff
@@ -179,8 +207,8 @@ def test_module_summary_with_depth(self):
 
     table = _get_module_table(
       module, 
-      {"dropout":random.PRNGKey(0), "params": random.PRNGKey(1)}, 
-      depth=1,
+      {"dropout":random.PRNGKey(0), "params": random.PRNGKey(1)},
+      method=CNN.__call__, mutable=True, depth=1,
```

Comment:
Why did you add the `method` kwarg here? It should be the default right?

Reply:
I removed all defaults from this private `_get_module_table` function, however, `method` here could be set to `None` so that `__call__` is internally used by default.0